src/layer/marker/index.js

```diff
@@ -2,6 +2,48 @@ import {Icon} from './Icon.js';
 export {icon} from './Icon.js';
 import {IconDefault} from './Icon.Default.js';
 Icon.Default = IconDefault;
+
+for (const color of ['Pink', 'DarkBlue', 'Lila', 'Magenta', 'Red', 'Orange', 'Brown', 'Green', 'Mint']) {
+	Icon.Default[color] = IconDefault.extend({
+		options:{
+			className: `default-icon-color-${color.toLowerCase()}`
+		}
+	});
+}
+Icon.Default.Blue = IconDefault;
+
+/**
+ * @miniclass Colored default icons (Icon)
+ * @section
+ *
+ * `L.Icon.Default` has several static properties providing different coloured versions of the default icon.
+ *
+ * Usage is as: `L.marker(latlng, { icon: new L.Icon.Default.Green() })`
+ *
+ * Note that `L.Icon.Default.Blue` behaves the same as `L.Icon.Default`.
+ *
+ * @property Pink: L.Icon
+ * The default icon, but tinted pink
+ * @property Blue: L.Icon
+ * The default icon, but tinted blue (same as default)
+ * @property DarkBlue: L.Icon
+ * The default icon, but tinted dark blue
+ * @property Lila: L.Icon
+ * The default icon, but tinted lila
+ * @property Magenta: L.Icon
+ * The default icon, but tinted magenta
+ * @property Red: L.Icon
+ * The default icon, but tinted red
+ * @property Orange: L.Icon
+ * The default icon, but tinted orange
+ * @property Brown: L.Icon
+ * The default icon, but tinted brown
+ * @property Green: L.Icon
+ * The default icon, but tinted green
+ * @property Mint: L.Icon
+ * The default icon, but tinted bluish green
+ */
```

Comment:
Creative approach, but TBH I think this adds too many bytes for something that most wont use, it's already possible to use custom markers. Am I missing something?

Side note: OTOH if markers were `<svg>` instead of `<img>` (which honestly I thought was the intention of https://github.com/Leaflet/Leaflet/issues/8158 / https://github.com/Leaflet/Leaflet/issues/8756) all it would require would be the option to change attributes (`fill`, `color`, etc.)?

Reply:
No we don't use the svg markers because #8766 is not merged. 

In my eyes we should make some things simpler for the users else we will lose more and more to other libraries. One of the things which can be really helpful (and is searched a lot), is how to change the color. I think this few bytes are worth it.

I would like to use the `stroke` and `fill` but we include the icons as a image and not as a svg element. I don't know how to change the css-classes / styles over a `<img>` element